man/rustc.1

```diff
@@ -242,6 +242,52 @@ full debug info with variable and type information.
 \fBopt\-level\fR=\fIVAL\fR
 Optimize with possible levels 0\[en]3
 
+.SH ENVIRONMENT VARIABLES
```

Comment:
It may be worth adding a description to this section saying that some of these environment variables affect the output of the compiler where some affect the output of all rust programs linking to the standard library.